administrator/components/com_joomgallery/helpers/upload.php

```diff
@@ -407,6 +453,27 @@ protected function uploadSingles()
         continue;
       }*/
 
+      // check for overriding with meta data
+      $overridevalues = array(
```

Comment:
Soweit ich das gesehen habe, wird der nachfolgende Code für die verschiedenen Uploadmethoden immer wieder in gleicher Weise aufgerufen. Das schreit doch nach einer Methode.